libass/ass_types.h

```diff
@@ -242,9 +242,9 @@ typedef struct ass_track {
     int PlayResY;
     double Timer;
     int WrapStyle;
-    int ScaledBorderAndShadow;
-    int Kerning;
-    char *Language;
+    int ScaledBorderAndShadow; // 0 or 1 (boolean)
+    int Kerning; // 0 or 1 (boolean)
+    char *Language; // zero-terminated ISO-639-1 codes
```

Comment:
this can be null, right? also, singular "code"

Reply:
thanks, removed the plural. Yes, NULL is allowed here.